From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@aashishr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (aashishr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

bwren committed:
Log Analytics view filters

bwren committed:
rough draft for review

bwren committed:
fixes

saghorpa committed:
VLI Changes

saghorpa committed:
VLI Changes

saghorpa committed:
VLI Changes

bwren committed:
image

bwren committed:
image border

jeahyoun committed:
Add clicktale feature flag to index.md.

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into bryanla-msi-b2b

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

PatAltimore committed:
Add test drive quickstarts

PatAltimore committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into patricka-quickstarts-master

PatAltimore committed:
Add test drive quickstarts to TOC

PatAltimore committed:
Update metadata

saghorpa committed:
VLI doc updated with additional backup/restore

PatAltimore committed:
Acrolinx updates

PatAltimore committed:
Add missing screenshot

PatAltimore committed:
Acrolinx updates

parakhj committed:
Updating Google typo

SaloniSonpal committed:
updating install instructions for npgsql

TomArcherMsft committed:
inital put of tutorial

TomArcherMsft committed:
inital put of tutorial

TomArcherMsft committed:
acrolynx pass

TomArcherMsft committed:
acrolynx pass 2

TomArcherMsft committed:
changed author alias as original alias is not found

TomArcherMsft committed:
more copy edits

TomArcherMsft committed:
acrolynx pass 3

TomArcherMsft committed:
normalized article/tutorial usage in article

ealsur committed:
Merge pull request #3 from MicrosoftDocs/master

Updating fork

ealsur committed:
Updating Change Feed changelog for version 1.2.0

ealsur committed:
Updating publish date

PatAltimore committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into patricka-quickstarts-master

TomArcherMsft committed:
changing west to east per pg request

TomArcherMsft committed:
corrected capitalization of HashiCorp

saghorpa committed:
VLI doc updated with additional backup/restore

saghorpa committed:
VLI doc updated with additional backup/restore1

TomArcherMsft committed:
added new article to toc

saghorpa committed:
VLI doc updated with additional backup/restore1

rayne-wiselman committed:
updating freshness and adding missing files to TOC

TomArcherMsft committed:
set author to VJ

twooley committed:
updated relnotes article for ASDK

barbkess committed:
update DWU explanations

twooley committed:
updated relnotes article for ASDK

bwren committed:
header

twooley committed:
updated relnotes article for ASDK

bwren committed:
Merge branch 'master' into la-view-filters

bwren committed:
toc

PatAltimore committed:
Add feedback

bwren committed:
Merge branch 'la-view-filters' of https://github.com/bwren/azure-docs-pr into la-view-filters

bwren committed:
toc

twooley committed:
updated relnotes article for ASDK

saghorpa committed:
Merge branch 'master' into VLIUpdate10062017

twooley committed:
updated relnotes article for ASDK

twooley committed:
updated relnotes article for ASDK

rayne-wiselman committed:
Update site-recovery-support-matrix-to-sec-site.md

rayne-wiselman committed:
Update site-recovery-migrate-to-azure.md

rayne-wiselman committed:
Update site-recovery-support-matrix-to-sec-site.md

spelluru committed:
On-prem SQL -> Azure Blob PowerShell sample

spelluru committed:
Cosmos DB -> Data Factory

spelluru committed:
removed highlighting

spelluru committed:
cleanup and added details

BryanApellanes committed:
custom-search: sanitize image

TomArcherMsft committed:
edits per review by pr reviewer

BryanLa committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into bryanla-msi-stuart-datalake

BryanLa committed:
updates

MGoedtel committed:
updated article per PM team

Commit:
errcodes

parakhj committed:
Random edit

BryanLa committed:
updates

rayne-wiselman committed:
Update site-recovery-overview.md

rayne-wiselman committed:
Update site-recovery-support-matrix-to-sec-site.md

adegeo committed:
dotnet install support for 4.7.1

PRMerger10 committed:
Merge pull request #26429 from parakhj/master

Updating Google typo

rayne-wiselman committed:
Update site-recovery-capacity-planner.md

Thraka committed:
Update cloud-services-dotnet-install-dotnet.md

rayne-wiselman committed:
Update site-recovery-capacity-planning-for-hyper-v-replication.md

rayne-wiselman committed:
Merge branch 'master' into rayne-october30-3

ktoliver committed:
Merge pull request #26541 from twooley/twasdkrelnotes

updated relnotes article for ASDK

v-shils committed:
Merge pull request #26575 from brapel/custom-search-fix-hh-image

custom-search: sanitize image

PRMerger17 committed:
Merge pull request #26584 from MarkusVi/errcodes

errcodes

cherylmc committed:
Update vpn-gateway-about-skus-legacy.md

spelluru committed:
fixed broken links

Lisaco88 committed:
Merge pull request #26457 from TomArcher/terraform-tutorial

Terraform tutorial

spelluru committed:
added to the main topic

carolinacmoravia committed:
M13837: Inconsistent term used 

Hello, @thraka,

Translator has reported possible source content issue. As this string is shown in the Imports (an element) section, assumed the string should read "only one imports block" instead of "runtime block."

Please, check the proposed file change and help to merge if agreed. If not, please, let me know either if I should close this PR, if you would like me to fix it in another way within this PR or if you prefer to fix it in another PR. Could you indicate your PR number in that case so I can check the new version, please?

Many thanks in advance.

spelluru committed:
spell error

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hybridpssample1031

spelluru committed:
availabile -> avialable

PRMerger17 committed:
Merge pull request #26590 from carolinacmoravia/patch-420

M13837: Inconsistent term used

PatAltimore committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into patricka-quickstarts-master

spelluru committed:
ready for publishing

barbkess committed:
fix typo

v-shils committed:
Merge pull request #26555 from saghorpa/VLIUpdate10062017

Vli update10302017

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hybridpssample1031

PRMerger12 committed:
Merge pull request #26591 from spelluru/availabile1031

availabile -> avialable

spelluru committed:
availabile -> available

bwren committed:
fixes

fhokholdMSFT committed:
Update key-vault-hsm-protected-keys.md

Updated date field

MGoedtel committed:
updated article per feedback

Commit:
Fixing note

Lisaco88 committed:
Merge pull request #26586 from Thraka/cs

dotnet install support for 4.7.1

PatAltimore committed:
Add reviewer feedback

cherylmc committed:
Update vpn-gateway-about-skus-legacy.md

PRMerger13 committed:
Merge pull request #26593 from spelluru/availabile1031-2

availabile -> available

spelluru committed:
acrolynx issues

MGoedtel committed:
updated note formatting

PRMerger10 committed:
Merge pull request #26597 from JSeb225/patch-15

Fixing note

v-shils committed:
Merge pull request #24788 from bwren/la-view-filters

Log Analytics view filters

PRMerger19 committed:
Merge pull request #26595 from fhokholdMSFT/patch-6

Update key-vault-hsm-protected-keys.md

Lisaco88 committed:
Merge pull request #26583 from MGoedtel/UpdateMultiVm10312017

updated article per PM team

ealsur committed:
Reflecting final publish date

cherylmc committed:
Update vpn-gateway-about-skus-legacy.md

SaloniSonpal committed:
updating install instructions for npgsql

Ja-Dunn committed:
Merge pull request #25508 from BryanLa/bryanla-msi-stuart-datalake

MSI datalake tutorials

SaloniSonpal committed:
updating install instructions for npgsql after merge

Ja-Dunn committed:
Merge pull request #26314 from PatAltimore/patricka-quickstarts-master

Add test drive quickstarts to B2C

JasonWHowell committed:
Update concepts-high-availability.md

Ja-Dunn committed:
Merge pull request #26517 from rayne-wiselman/rayne-october30-3

updating freshness and adding missing files to TOC

Ja-Dunn committed:
Merge pull request #26573 from spelluru/hybridpssample1031

On-prem SQL -> Azure Blob PowerShell sample

Ja-Dunn committed:
Merge pull request #26605 from JasonWHowell/patch-8

MySQL: Update concepts-high-availability.md

PRMerger15 committed:
Merge pull request #26545 from barbkess/jrj3

update DWU explanations

Commit:
freshness02

Commit:
Merge pull request #26607 from MarkusVi/freshness02

freshness02

Commit:
repapi02

Ja-Dunn committed:
Merge pull request #26599 from ealsur/master

Change Feed Processor SDK 1.2.0 release notes

Commit:
Merge pull request #26442 from SaloniSonpal/master

Updating install instructions for npgsql

tynevi committed:
Merge pull request #25302 from jeahyoun/1097978-clicktale

Add clicktale feature flag to index.md.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

MicrosoftGuyJFlo committed:
Fix H2 issue

Ja-Dunn committed:
Merge pull request #26615 from MarkusVi/repapi02

repapi02

PRMerger20 committed:
Merge pull request #26589 from cherylmc/patch-1

Update vpn-gateway-about-skus-legacy.md

PRMerger15 committed:
Merge pull request #26619 from MicrosoftGuyJFlo/patch-4

Fix H2 issue

v-alje committed:
Merge pull request #26616 from MicrosoftDocs/FromPublicRepo

From public repo
